WooCommerce/Classes/ViewRelated/Orders/Order Filters/Date Range Filter/Custom Range Filter/DateRangeFilterViewController.swift

```diff
@@ -0,0 +1,231 @@
+import UIKit
+
+// MARK: - DateRangeFilterViewController
+// Allow to filter the orders by a range of dates
+//
+final class DateRangeFilterViewController: UIViewController {
```

Comment:
nice! I think this view controller can be used for any date range picker use cases

Reply:
Absolutely. I have made the comment more generic.